x-pack/plugins/observability_solution/apm/server/routes/traces/get_trace_items.ts

```diff
@@ -133,8 +149,29 @@ export async function getTraceItems({
 
   const traceDocsTotal = traceResponse.total;
   const exceedsMax = traceDocsTotal > maxTraceItems;
-  const traceDocs = traceResponse.hits.map((hit) => hit._source);
-  const errorDocs = errorResponse.hits.hits.map((hit) => hit._source);
+
+  const traceDocs = traceResponse.hits.map(({ hit }) => hit);
```

Comment:
Q: Should we `unflattenKnownApmEventFields` here as well 🤔

Reply:
`traceResponse` is assigned with what `getTraceDocsPaginated` returns. `getTraceDocsPaginated` is responsible for unflattening `fields`.